src/builtins/BuiltinSharedArrayBuffer.cpp

```diff
@@ -111,17 +142,31 @@ void GlobalObject::installSharedArrayBuffer(ExecutionState& state)
                                                                    ObjectPropertyDescriptor(Value(strings->SharedArrayBuffer.string()), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::ConfigurablePresent)));
 
     {
```

Comment:
IMO,  `{ }` is no longer needed as you gave each one a unique name.

Reply:
Yes, right.
It depends on one's preference I think,
I wrapped this part with a brace pair because `JSGetterSetter` is temporally allocated as getter/setter holder, so I wanted to deallocate it immediately right after its usage. In addition, it looks tidy code for me (at least).